heron/instance/src/java/com/twitter/heron/instance/spout/SpoutInstance.java

```diff
@@ -186,8 +186,6 @@ public void init(State<Serializable, Serializable> state) {
   public void start() {
     // Add spout tasks for execution
     addSpoutsTasks();
-
-    topologyState = TopologyAPI.TopologyState.RUNNING;
```

Comment:
this is expected?

Reply:
no need to use `topologyState ` any more